src/utils/config.js

```diff
@@ -2,10 +2,9 @@ import getQueryParam from 'utils/getQueryParam';
 
 const develop01 = {
   authOptions: {
-    // TODO switch to single endpoint
-    accountsUrl: `https://accounts-develop01.develop.virtru.com`,
-    acmUrl: `https://acm-develop01.develop.virtru.com`,
-    apiUrl: `https://api-develop01.develop.virtru.com`,
+    accountsUrl: 'https://api-develop01.develop.virtru.com/accounts',
+    acmUrl: 'https://api-develop01.develop.virtru.com/acm',
+    apiUrl: 'https://api-develop01.develop.virtru.com',
   },
   clientConfig: {
     auditEndpoint: 'https://audit-develop01.develop.virtru.com',
```

Comment:
audit also? 'https://api-develop01.develop.virtru.com/audit'

Reply:
Audit wasn't exposed via API when I was doing this.  lemme try it and see what happens